blog/2015-03-31-Eclipse_Docker_Tooling.adoc

```diff
@@ -0,0 +1,97 @@
+
+Built-in filters can show/hide intermediate and 'dangling' images as well as
+stopped containers.
+
+== Managing Docker Images
+
+The *Docker Images view* lists all images in the Docker host selected in the
+Docker Explorer view. This view allows user to manage images, including:
+
+- Pulling images from the https://registry.hub.docker.com/[Docker Hub Registry] (other registries will be supported as well)
+- Uploading images to the https://registry.hub.docker.com/[Docker Hub Registry]
+- Building images from a Docker files
+- Running an image
+
+image::images/docker_tooling_images_view.png[Docker Images view]
+
+A wizard let the user input all the arguments to create a new container from an image.
+When the container is started, all the logs can be streamed into the Eclipse Console:
+
+image::images/docker_tooling_console.png[Docker Images view]
+
+== Managing Docker Containers
+
+The *Docker Containers* view lets the user manage her containers. The view toolbar
+provides commands to start, stop, pause, unpause, attach and kill containers.
+
+image::images/docker_tooling_containers_view.png[Docker Containers view]
+
+This view also provides a filter to show/hide stopped containers.
+Users can also attach an Eclipse console to a running Docker container to follow
+the logs and use the STDIN to interact with it.
+
+== Info and Inspect on Images and Containers
+
+We also integrate with the Eclipse Properties view to provide users with info and 'inspect'
+data about a selected container or image.
+
+image::images/docker_tooling_properties_view.png[Properties view]
+
+== Where is the code ?
+
+Roland Grunberg and Jeff Johnston from Red Hat started this project and are currently
+in the process of getting the code accepted to Eclipse.org as part of the
+http://www.eclipse.org/linuxtools/[Linux Tools project] (even if the tooling also runs on other platforms), which explains
+why the code has not been made public yet.
+
+Taking about code, we rely on the https://github.com/spotify/docker-client[open source Docker client
+developed by Spotify] to handle the low-level communication with the Docker daemons
+and I should thank the Spotify developers who maintain this library.
+They've been very kind to quickly review and merge the pull requests
+that we've submitted and it's been a pleasure to contribute to their project.
+Open source collaboration FTW :-)
+
+== What's next ?
+
+There is still some work and the screenshots showed above may still
+evolve as we add more features, but we hope that this blog entry will give
+you a taste of what's coming soon in Eclipse with regards to Docker tooling.
+
+While the code will be hosted in http://www.eclipse.org/linuxtools/[Linux Tools project]
+at Eclipse.org, we intend to also ship it as part of
+JBoss Developer Studio 9 and JBoss Tools 4.3 later this year.
+
+We also created a https://issues.jboss.org/browse/JBIDE/component/12321304[JIRA component] to track this work-in-progress in the scope of JBoss Tools.
```

Comment:
link to jira https://issues.jboss.org/browse/JBIDE-19496 and https://issues.jboss.org/browse/JBIDE-19498 

...and I actually think we are missing some in that 'docker' component ;)